cli/run-export-test.js

```diff
@@ -52,11 +52,11 @@ else {
   });
 }
 
-const plugin = plugins[args.plugin];
-const files = plugin.export.export(fixtures, {});
+const plugin = require(path.join(__dirname, `../plugins`, args.plugin, `export.js`));
```

Comment:
Create a reusable variable ``const pluginDir = path.join(__dirname, `../plugins`, args.plugin)``?

Reply:
Doesn't make too much sense here since both `plugin` and `test` would still need to use `path.join(pluginDir, ...)`.